ibc_public/utils_labels.py

```diff
@@ -0,0 +1,162 @@
+import json
+import sys
+import warnings
+
+import numpy as np
+import pandas as pd
+
+import os
+
+_package_directory = os.path.dirname(os.path.abspath(__file__))
+
+ALL_CONTRASTS = os.path.join(_package_directory, '..', 'ibc_data',
+                             'all_contrasts.tsv')
+
+
+def get_labels(contrasts='all'):
+    """
+    Returns the list of labels for each passed contrast name
+
+    Parameters
+    ----------
+
+    contrasts: list of str, default 'all'
+               Each element of the list is a contrast name in the document.
+               The default argument will select all present contrasts
+
+    Returns
+    -------
+
+    contrast_dict: dict
+                   Dictionary containing the contrasts provided by the user
+                   as keys, and their corresponding labels as values
+    """
+    df = pd.read_csv(ALL_CONTRASTS, sep='\t')
+    if contrasts == 'all':
+        contrasts = df['contrast'].values
+
+    contrast_dict = {}
+
+    con_slice = df[df['contrast'].isin(contrasts)]
+    not_found = np.setdiff1d(contrasts, con_slice['contrast'])
+
+    if not_found.size != 0:
+        warnings.warn(f"The following contrast names were not found: "
+                      f"{not_found}")
+
+    for index, con in con_slice.iterrows():
+
+        labels = con.loc[con == 1.0].index
+        con_name = f"({con.task}) {con.contrast}"
+        contrast_dict[con_name] = [label for label in labels]
+
+    return contrast_dict
+
+
+def _binary_question(question):
```

Comment:
This can be removed as far as I can see.

Reply:
Removed